hedera-node/hedera-app/src/main/java/com/hedera/node/app/HederaInjectionComponent.java

```diff
@@ -92,6 +93,8 @@ public interface HederaInjectionComponent {
 
     RecordCache recordCache();
 
+    HederaRecordCache hederaRecordCache();
```

Comment:
Do we need these two lines? I think what we have in `HederaStateInjectionModule` should be sufficient?

Reply:
I added them for consistency but you are right - they not used